scylla-cdc/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "scylla-cdc"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
I'll have to bump the version to `0.2.0` before releasing. Some types from the driver are exposed in the API of this library, and `0.8.1` -> `0.13.1` is not a semver-compatible change.